src/api/v1/metadata.ts

```diff
@@ -56,10 +65,8 @@ metadataRouter.get(
 
       res.json(metadata);
     } catch (error) {
-      console.error("Failed to fetch or create conversation metadata:", error);
```

Comment:
An alternative is to simply remove the `try/catch` and let the error middleware catch the exception. It should do the right thing